src/main/java/com/SpringBootProject/DaoServices/ClientsDaoService.java

```diff
@@ -0,0 +1,187 @@
+package com.SpringBootProject.DaoServices;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import com.SpringBootProject.Entity.Clients;
+
+
+public class ClientsDaoService implements ClientsDaoInterface {
+	private static String URL = "jdbc:mysql://localhost:3306/TestProject";
+	private static String USERNAME = "root";
+	private static String PASSWORD = "0905";
+	private Connection connection = CreateConnection();
+	
+	
+	private static final Logger logger = Logger.getLogger(ClientsDaoService.class);
+	
+	private static String QUERY_GET_ALL_CLIENTS = "{SELECT id, SurName, Name, Passport, HomeAddress, Telephone FROM `Client`}";
+	private static String QUERY_SELECT_CLIENT_BY_ID = "{SELECT id, SurName, Name, Passport, HomeAddress, Telephone  FROM `Client` WHERE id= :id}";
+	private static String QUERY_CREATE_CLIENT = "{INSERT INTO `Client`(SurName, Name, Passport, HomeAddress, Telephone) VALUES(:Sname, :name, :passport, :homeaddress, :telephone)}";
+	private static String QUERY_UPDATE_CLIENT_BY_ID = "{UPDATE `Client` SET SurName= :Sname, Name= :name, Passport= :passport, HomeAddress= :homeaddress, Telephone= :telephone WHERE id= :id}";
+	private static String QUERY_DELETE_CLIENT = "{DELETE FROM `Client` WHERE id= :id}";
+		
+	/**
+	 * This method create and return a connection
+	 * @return Connection
+	 */
+	private Connection CreateConnection(){
+		try {
+			return DriverManager.getConnection(URL, USERNAME, PASSWORD);
+		}
+		catch(SQLException e){
+			logger.error(e.getMessage());
```

Comment:
Instead of `e.getMessage()` use just `e` as if you will be having exception here you won't  know cause.